app/Http/Requests/StorePostRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StorePostRequest extends FormRequest
```

Comment:
Please rename to StoreRequest.
We usually put this in subdirectories based on the user role then model.
So for example,
App\Http\Requests\User\Post\StoreRequest;
App\Http\Requests\Admin\Post\StoreRequest;